packages/server/modules/workspacesCore/helpers/graphTypes.ts

```diff
@@ -0,0 +1,3 @@
+import { Workspace } from '@/modules/workspacesCore/domain/types'
+
+export type WorkspaceRecordGraphQLReturn = Workspace
```

Comment:
This doesn't appear to be used in any way. The purpose for these mappings is to use them in codegen.yml to configure what various returns are supposed to be in GQL types of resolvers

Reply:
+ The name should be `WorkspaceGraphQLReturn`, the `Record` bit comes out of nowhere. I'd kinda get it if `Workspace` would be called `WorkspaceRecord`, but if it isn't called that then it doesn't make sense to add it in this type name.

Reply:
resolved as discussed!